ui/components/HelloApp.vue

```diff
@@ -17,20 +17,22 @@
 </template>
 
 <script>
-import { mapGetters, mapActions } from 'vuex'
-
 export default {
   name: 'HelloApp',
   data: function () {
     return {
       name: 'World'
     }
   },
-  methods: {
-    ...mapActions('Hello', ['submitName'])
-  },
   computed: {
-    ...mapGetters('Hello', ['message'])
+    message () {
+      return this.$store.getters['Hello/message']
```

Comment:
Why? These vuex helpers are pretty common...

Reply:
Then we would have to bundle vuex into the final bundle. This way we don't have to bundle it and haven't got the same amount of vuex dependencies as extensions.

Reply:
But if there is a dead code detection it will remove all unused vuex features, right?